plato/draw/Box.py

```diff
@@ -0,0 +1,56 @@
+import itertools
+import numpy as np
+from .Lines import Lines
+from .. import math
+from .internal import attribute_setter, ShapeDecorator, ShapeAttribute
+
+@ShapeDecorator
+class Box(Lines):
+    """Create a box primitive to draw the frame
+
+    Example: prim = draw.Box((Lx,Ly,Lz,xy,xz,yz), width, color)
+    
+    """
+
+    def __init__(self, *args, **kwargs):
+        # argument order: box, width, color
+        # Box tuple: (Lx, Ly, Lz, xy, xz, yz)
+        box, width, color = None, None, None
```

Comment:
I'm not sure what the right way to deal with these arguments is, but this is rather clunky. @klarh probably has a good idea.

Reply:
We still haven't figured out the best way to manage args.